packages/manager/modules/web-paas/src/details/service/add-addon/template.html

```diff
@@ -0,0 +1,46 @@
+<form
+    id="addAddon"
+    name="addAddon"
+    data-ng-submit="$ctrl.addon.quantity > 0 && $ctrl.orderAddon()"
+    novalidate
+>
+    <oui-modal
+        data-primary-label="{{ ::'web_paas_common_confirm' | translate }}"
+        data-secondary-action="$ctrl.goBack()"
+        data-secondary-label="{{ ::'web_paas_common_cancel' | translate }}"
+        data-on-dismiss="$ctrl.goBack()"
+        data-primary-disabled="$ctrl.disableSubmit"
+        data-heading="{{ ::'web_paas_service_add_addon_title_' + $ctrl.addonType.split('-').pop() | translate }}"
```

Comment:
refer below comment. having a class will avoid this kind of duplicate and complex logic in template.